js/bytex.js

```diff
@@ -909,76 +1020,61 @@ module.exports = class bytex extends Exchange {
         //     }
         //
         const marketId = this.safeString (order, 'symbol');
+        const symbol = this.safeSymbol (marketId, market, '-');
+        const id = this.safeString (order, 'id');
         const timestamp = this.parse8601 (this.safeString (order, 'created_at'));
-        const meta = this.safeValue (order, 'meta');
-        const filled = this.safeString (order, 'filled');
+        const type = this.safeString (order, 'type');
+        const side = this.safeString (order, 'side');
+        const price = this.safeString (order, 'price');
         const amount = this.safeString (order, 'size');
+        const filled = this.safeString (order, 'filled');
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
```

Comment:
I think it's a better idea to just place the variables within the response object if the variables aren't reused, it reduces the amount of code